src/sharedInterfaces/queryResult.ts

```diff
@@ -76,7 +70,7 @@ export interface QueryResultWebviewState extends ExecutionPlanWebviewState {
     actualPlanEnabled?: boolean;
     selection?: ISlickRange[];
     executionPlanState: ExecutionPlanState;
-    filterState: Record<string, ColumnFilterState>;
+    filterState: Record<string, Record<string, ColumnFilterState>>;
```

Comment:
docstring explaining what's going on with the nested records (what are the first- and second-order keys?)

Reply:
This will be removed actually because I'm moving the filterState outside of the react state